spec/models/condition_spec.rb

```diff
@@ -1,4 +1,24 @@
 describe Condition do
+  context ".seed" do
```

Comment:
Minor, but prefer `describe ".seed" do` over a `context` in this case.  It's not a real official pattern yet, but most of the RSpec tests are moving that way.